### PR TITLE
Make all cli functions and structs crate-private

### PR DESCRIPTION
**Pull request by 0xPoe**
part of #2730

### CODE REVIEW

**Comment by 0xPoe** [comment]
@kinnison Could you please take a look? Thanks!

**Comment by rbtcollins** [comment]
@hi-rustin have you checked that there is no lint on Windows builds?

**Comment by 0xPoe** [comment]
> @hi-rustin have you checked that there is no lint on Windows builds?

I checked the CI relevant build log and found no warnings, do I still need to check it manually?

**Comment by 0xPoe** [comment]
@rbtcollins @kinnison Do you have time to help look at it? Thanks!

File: src/cli/markdown.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@ struct LineWrapper<'a, T: Terminal> {
     indent: u32,
     margin: u32,
     pos: u32,
-    pub w: &'a mut T,
+    pub(crate) w: &'a mut T,
```
Comment (rbtcollins) [inline src/cli/markdown.rs]: this is a public attribute in a private struct. What happens if you just make the attribute non-public?

Comment (0xPoe) [reply src/cli/markdown.rs]: Addressed. Good catch! Thank you.

**Pull request merged**